### PR TITLE
CRDs are related to KAS because people ask why their resources aren't…

### PR DESCRIPTION
**Pull request by deads2k**
… present

Some bugs have come in recently and we need to have CRD manifests for SCC, CRQ, and servicemonitor.

/assign @sttts

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -123,6 +123,7 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		"kube-apiserver",
 		[]configv1.ObjectReference{
 			{Group: "operator.openshift.io", Resource: "kubeapiservers", Name: "cluster"},
+			{Group: "apiextensions.k8s.io", Resource: "customresourcedefinitions"},
```
Comment (sttts) [inline pkg/operator/starter.go]: what does this mean without name?

Comment (deads2k) [reply pkg/operator/starter.go]: > what does this mean without name?

collect all of them.

**Comment by deads2k** [comment]
not as pretty as one may hope, but it worked: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/688/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws/3118/artifacts/e2e-aws/must-gather/registry-svc-ci-openshift-org-ci-op-j0fi4b6m-stable-sha256-c31e3068a603b8d8add473dbaaa5b933323a23dc862cb266855248e7cba5ac99/cluster-scoped-resources/apiextensions.k8s.io/customresourcedefinitions/

**Pull request merged**